app/src/main/res/layout-land/profile_chooser_profile_view.xml

```diff
@@ -12,7 +12,7 @@
     <import type="android.view.View" />
 
     <variable
-      name="wasProfileEverBeenAddedValue"
+      name="hasProfileEverBeenAddedValue"
```

Comment:
Is this something that's from a parent branch? I would have expected this needed to change everywhere in an earlier PR.

Reply:
Yes, I believe so. The diff history has been really off with most of my PRs and I am not sure why.